homeassistant/components/notion/__init__.py

```diff
@@ -194,8 +197,8 @@ def __init__(
     def available(self) -> bool:
         """Return True if entity is available."""
         return (
```

Comment:
```
    @property
    def available(self) -> bool:
        """Return True if entity is available."""
        return super().available and self._task_id in self.coordinator.data["tasks"]
```

Then, if the availability "algorithm" for `CoordinatorEntity` ever changes, this won't need changing, too?